src/keys/rsa-browser.js

```diff
@@ -120,3 +120,35 @@ function derivePublicFromPrivate (jwKey) {
     ['verify']
   )
 }
+
+/*
+
+RSA encryption/decryption for the browser with webcrypto workarround
+"bloody dark magic. webcrypto's why."
+
+Explanation:
+  - Convert JWK to PEM
+  - Load PEM with nodeForge
+  - Convert msg buffer to nodeForge buffer: ByteBuffer is a "binary-string backed buffer", so let's make our buffer a binary string
+  - Convert resulting nodeForge buffer to buffer: it returns a binary string, turn that into a uint8array(buffer)
+
+*/
+
+const forge = require('node-forge')
+const pki = forge.pki
```

Comment:
Passing in the module as an extension sounds like a good approach, at least for now, so we can get this released.

Also, changing this to `const pki = require('node-forge/lib/pki')` will also shave off some kb, but the majority comes from `pem-jwk`.

Reply:
Just found this: https://github.com/libp2p/js-libp2p-crypto/blob/master/src/keys/rsa.js#L28-L29

Also, applied that change and removed the PEM intermediary step for converting JWK to Forge

Reply:
https://github.com/libp2p/js-libp2p-crypto/pull/125#issuecomment-546312897